og.og_permissions.yml

```diff
@@ -2,10 +2,11 @@ update group:
   title: 'Edit group'
   description: 'Edit the group. Note: This permission controls only node entity type groups.'
   'default role':
-    - OG_ADMINISTRATOR_ROLE
```

Comment:
Yes. That's is pretty much what the extension discovery does I think. So
let's do that.

Yeah, that's why we can't really just use those global constants. We can
add them to the Og or another class though.
On 29 Nov 2015 18:03, "Amitai Burstein" <notifications@github.com> wrote:

> In og.og_permissions.yml
> <https://github.com/amitaibu/og/pull/48#discussion_r46095668>:
>
> > @@ -2,10 +2,11 @@ update group:
> >    title: 'Edit group'
> >    description: 'Edit the group. Note: This permission controls only node entity type groups.'
> >    'default role':
> > -    - OG_ADMINISTRATOR_ROLE
>
> Any pointer to those 'special' strings?
>
> Oh, we could just process them ourself in
> \Drupal\og\OgPermissionHandler::buildPermissionsYaml - is that what you
> meant? Anyway, we'll still have re-define the constants in the test file,
> for the unit test to pass
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/48/files#r46095668>.
>